ui/tests/integration/components/identity/item-details-test.js

```diff
@@ -0,0 +1,56 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import { create } from 'ember-cli-page-object';
+import itemDetails from 'vault/tests/pages/components/identity/item-details';
+import Ember from 'ember';
+
+const component = create(itemDetails);
+const { getOwner } = Ember;
+
+moduleForComponent('identity/item-details', 'Integration | Component | identity/item details', {
+  integration: true,
+  beforeEach() {
+    component.setContext(this);
+    getOwner(this).lookup('service:flash-messages').registerTypes(['success']);
+  },
+  afterEach() {
+    component.removeContext();
+  }
+});
+
+test('it renders the disabled warning', function(assert) {
+  let model = Ember.Object.create({
+    save() {
+      return Ember.RSVP.resolve();
+    },
+    disabled: true,
+    canEdit: true
+  });
+  sinon.spy(model, 'save');
+  this.set('model', model);
+  this.render(hbs`{{identity/item-details model=model}}`);
+  assert.dom('[data-test-disabled-warning]').exists();
```

Comment:
QUnit DOM looks really helpful.

Reply:
It's been really nice where we don't want / need / already have a full ember-cli-page object.